packages/components/src/components/combobox/component.tsx

```diff
@@ -0,0 +1,410 @@
+import type {
+	HideErrorPropType,
+	IdPropType,
+	InputTypeOnDefault,
+	KoliBriHorizontalIcons,
+	LabelWithExpertSlotPropType,
+	MsgPropType,
+	NamePropType,
+	OptionsWithOptgroupPropType,
+	RowsPropType,
+	ComboBoxAPI,
+	SelectOption,
+	ComboBoxStates,
+	Stringified,
+	SyncValueBySelectorPropType,
+	TooltipAlignPropType,
+	W3CInputValue,
+} from '@public-ui/schema';
+import { Component, Element, h, Host, Method, Prop, State, Watch, Fragment } from '@stencil/core';
+
+import { nonce } from '../../utils/dev.utils';
+import { stopPropagation, tryToDispatchKoliBriEvent } from '../../utils/events';
+import { SelectController } from './controller';
+
+import type { JSX } from '@stencil/core';
+import { KolIconTag, KolInputTag } from '../../core/component-names';
+import { showExpertSlot } from '@public-ui/schema';
+import { InternalUnderlinedAccessKey } from '../span/InternalUnderlinedAccessKey';
+// const isSelected = (valueList: unknown[] | null, optionValue: unknown): boolean => {
+// 	return Array.isArray(valueList) && valueList.includes(optionValue);
+// };
+
+/**
+ * @slot - Die Beschriftung des Eingabefeldes.
+ */
+@Component({
+	tag: 'kol-combo-box',
+	styleUrls: {
+		default: './style.scss',
+	},
+	shadow: true,
+})
+export class KolCombobox implements ComboBoxAPI {
+	@Element() private readonly host?: HTMLKolComboBoxElement;
+	private ref?: HTMLSelectElement;
+
+	@Method()
+	// eslint-disable-next-line @typescript-eslint/require-await
+	public async getValue(): Promise<Stringified<W3CInputValue[]> | undefined> {
+		return this._value;
+	}
+
+	private toggleListbox = () => {
+		this._isOpen = !this._isOpen;
+	};
+	private selectOption(option: SelectOption<W3CInputValue>) {
+		this.state._inputValue = option.label;
+		this.toggleListbox();
+	}
+	private OnInput(event: Event) {
+		const target = event.target as HTMLInputElement;
+		this.state._inputValue = target.value;
+		this.filtredOptions(this.state._inputValue);
+	}
+
+	private filtredOptions(query: string) {
```

Comment:
```suggestion
	private setFilteredOptionsByQuery(query: string) {
```
Typo in `filtered`, außerdem bitte klarer benennen (was tut die Methode). Vorschlag siehe oben.